web/tests/test_chapters.py

```
@@ -0,0 +1,292 @@
+import datetime
+
+from django.test import TestCase, Client
+from django.urls import reverse
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.core.files.uploadedfile import SimpleUploadedFile
```

Ruff (0.8.2)

`django.core.files.uploadedfile.SimpleUploadedFile` imported but unused

Remove unused import: `django.core.files.uploadedfile.SimpleUploadedFile`

(F401)